src/test_workflow/benchmark_test/benchmark_test_runner.py

```diff
@@ -38,3 +39,17 @@ def run_tests(self) -> None:
 
     def run(self) -> None:
         self.run_tests()
+
+    def get_git_ref(self) -> str:
+        if self.test_manifest:
+            os_major_version = self.test_manifest.version.split(".")[0]
+            if os_major_version in ['2', '3']:
+                return 'main'
```

Comment:
Is there a better way of handling this?
Hardcoding 2/3 as main branch feels hard to maintain over time.
Would appreciate a version property file or manifest to retrieve latest version vs others.
Or any other way of not hardcoding.

Reply:
The cluster cdk branch is tightly coupled to OpenSearch version, even if we move it to a property file or as such, this check would still be required to figure out for which version of OpenSearch we are running the benchmark against. 

Also, I do not see this changing in near future as we only have 3 major versions to maintain out of which 3.x is yet to be released next year and hopefully will be in development for next couple of years till 4.x comes along.

Reply:
Once we end support for 1.x, this whole logic can be removed, assuming that there will not be breaking change with respect to cluster bootstrapping.

Reply:
This could have been ` if os_major_version not in ['1']:` but since we have been asked to run becnhmark against 7.10.2 it becomes ` if os_major_version not in ['1', '7']:` which then doesn't make it better.

Reply:
Can you open an issue to track this removal of 1.x?